rasa_nlu/evaluate.py

```diff
@@ -24,17 +25,22 @@
                              "ordinal", "phone-number", "timezone", "temperature", "time", "url", "volume"}
 entity_processors = {"ner_synonyms"}
 
-
 def create_argparser():  # pragma: no cover
+    import argparse
     parser = argparse.ArgumentParser(
-            description='evaluate a trained Rasa NLU pipeline')
+            description='evaluate a Rasa NLU pipeline with cross validation or on external data')
 
-    parser.add_argument('-d', '--data', default=None,
-                        help="file containing evaluation data")
-    parser.add_argument('-m', '--model', required=True,
-                        help="path to model")
+    parser.add_argument('-d', '--data', required=True,
+                        help="file containing training/evaluation data")
+    parser.add_argument('--mode', required=True,
```

Comment:
let's make the `evaluation` mode the default mode (for backwards compatibility) and make this flag optional (as well as remove the flag from the evaluation example)